README.md

```diff
@@ -379,9 +379,9 @@ current date, bumped (minor) version, and commit message.
 If `CHANGELOG.md` is absent in the repository, nothing will happen.
 
 
-#### Defining templates
+#### Working with templates
 
-As commented, files within moduleroot directory can be flat files or ERB templates. These files have direct access to @configs hash, which gets values from config_defaults.yml file and from the module being processed:
+As mentioned, files in the moduleroot directory are ERB templates (they must have an .erb extension, or they will be ignored). These files have direct access to @configs hash, which gets values from config_defaults.yml file and from the module being processed:
```

Comment:
`must be` would be better than `are` here.

Reply:
Done.